modules/registry-exposing-secure-registry-manually.adoc

```diff
@@ -80,3 +80,6 @@ spec:
 Only set `secretName` if you are providing a custom TLS configuration for the
 registry's route.
 ====
+
+.Troubleshooting
+* link:https://access.redhat.com/solutions/5419501[Error creating TLS secret]
```

Comment:
In build: https://62748--docspreview.netlify.app/openshift-rosa/latest/registry/securing-exposing-registry.html#registry-exposing-secure-registry-manually_securing-exposing-registry